plugin.video.retrospect/channels/channel.videos/twit/chn_twit.py

```diff
@@ -74,62 +86,11 @@ def add_live_stream(self, data):
 
         items = []
 
-        item = MediaItem("\a.: TWiT.TV Live :.", "http://live.twit.tv/")
-        item.complete = True
-
-        playback_item = MediaItem("Play Live", "http://live.twit.tv/")
-        playback_item.media_type = mediatype.VIDEO
-        playback_item.isLive = True
-        playback_part = playback_item.create_new_empty_media_part()
-
-        # noinspection PyStatementEffect
-        """
-        BitGravity
-        There are two streams available from BitGravity; a 512 kbps low-bandwidth stream
-        and a 1 Mbps high-bandwidth stream.
-
-        UStream
-        This is the default stream. The UStream stream is a variable stream that maxes at
-        2.2 Mbps and adjusts down based on your bandwidth.
-        Justin.tv
-
-        The Justin.tv stream is a 2.2 mbps high-bandwidth stream that will adjust to lower
-        bandwidth and resolutions.
-
-        Flosoft.biz
-        The Flosoft.biz stream is a 5 resolution/bitrate HLS stream, intended for our app developers.
-        Please see Flosoft Developer Section. This stream is hosted by TWiT through Flosoft.biz
-        """
-
-        # http://wiki.twit.tv/wiki/TWiT_Live#Direct_links_to_TWiT_Live_Video_Streams
-        media_urls = {
-            # Justin TV
-            # "2000": "http://usher.justin.tv/stream/multi_playlist/twit.m3u8",
-
-            # Flosoft (http://wiki.twit.tv/wiki/Developer_Guide#Flosoft.biz)
-            "264": "http://hls.cdn.flosoft.biz/flosoft/mp4:twitStream_240/playlist.m3u8",
-            "512": "http://hls.cdn.flosoft.biz/flosoft/mp4:twitStream_360/playlist.m3u8",
-            "1024": "http://hls.cdn.flosoft.biz/flosoft/mp4:twitStream_480/playlist.m3u8",
-            "1475": "http://hls.cdn.flosoft.biz/flosoft/mp4:twitStream_540/playlist.m3u8",
-            "1778": "http://hls.cdn.flosoft.biz/flosoft/mp4:twitStream_720/playlist.m3u8",
-
-            # UStream
-            "1524": "http://iphone-streaming.ustream.tv/ustreamVideo/1524/streams/live/playlist.m3u8",
-
-            # BitGravity
-            # "512": "http://209.131.99.99/twit/live/low",
-            # "1024": "http://209.131.99.99/twit/live/high",
-            #"512": "http://64.185.191.180/cdn-live-s1/_definst_/twit/live/low/playlist.m3u8",
-            #"1024": "http://64.185.191.180/cdn-live-s1/_definst_/twit/live/high/playlist.m3u8",
-        }
-
-        for bitrate in media_urls:
-            playback_part.append_media_stream(media_urls[bitrate], bitrate)
-
-        Logger.debug("Streams: %s", playback_part)
-        playback_item.complete = True
-        item.items.append(playback_item)
-        Logger.debug("Appended: %s", playback_item)
+        item = MediaItem("\a.: TWiT.TV Live :.",
+                         "http://iphone-streaming.ustream.tv/uhls/1524/streams/live/iphone/playlist.m3u8",
```

Comment:
Odd stream URL this one

Reply:
Indeed, but an official stream: https://wiki.twit.tv/wiki/TWiT_Live